portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -42,9 +56,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String text = getParameter(request, "comment-text", "");
     String commenterName = getParameter(request, "commenter-name", "Anonymous");
     String commenterEmail = getParameter(request, "commenter-email", "Unknown");
+    long timestamp = System.currentTimeMillis();
```

Comment:
nit: I think you can inline 'timestamp'.

Reply:
Done.